apps/test/unit/gamelab/spritelab/spriteCommandsTest.js

```diff
@@ -0,0 +1,157 @@
+/* global p5 */
+import {expect} from '../../../util/configuredChai';
```

Comment:
Use `../../../util/reconfiguredChai` since configuredChai contains a module that is being deprecated.